app/Http/Resources/NewsResource.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Http\Resources;
+
+use Illuminate\Http\Request;
+use Illuminate\Http\Resources\Json\ResourceCollection;
+
+class NewsResource extends ResourceCollection
+{
+    /**
+     * Transform the resource collection into an array.
+     *
+     * @return array<int|string, mixed>
+     */
+    public function toArray(Request $request): array
+    {
+        $coverPhoto = $this->getFirstMedia('cover_photos');
+
+        return [
+            'id' => $this->id,
+            'organisation' => [
+                'id' => $this->organisation->id ?? null,
+                'name' => $this->organisation->name ?? null,
```

Comment:
```suggestion
                'id' => $this->organisation->id,
                'name' => $this->organisation->name,
```

organizația nu e nullable în db, so it looks like we should always have it.